jest.config.js

```diff
@@ -21,6 +21,9 @@ module.exports = {
   transformIgnorePatterns: [
     '<rootDir>/node_modules/(?!(ol|antd|@babel|jest-runtime|(rc-*[a-z]*)|@ant-design|@terrestris))'
   ],
+  testPathIgnorePatterns: [
+    '<rootDir>/src/e2e-tests/'
```

Comment:
Is this needed? I would have expected that the e2e tests are skipped already (see `testMatch`).

Reply:
You are right. I added the `testPathIgnorePatterns` because of slightly different setup. But mentioning them in this setup is redundant.